### ISSUE TITLE
F-Droid release.

### ISSUE THREAD

**Issue by PoorPocketsMcNewHold** [Nekogram/Nekogram#24]
Hi there,
I've recently discovered your app, and it quickly became my favorite Telegram client during the past days, due to how innovative compared to other clients, Nekogram was.
However, two traces of trackers as been found in your app, by running a simple exodus scan with ClassyShark3xodus.
So, I'm redacting this GitHub issue, if you could provide tracker-free release builds (Atleast, in your GitHub repo) to be eligible to be released on the FOSS only, F-droid store.

**Comment by b3pio** [comment]
App (github version at least) contains Google and Microsoft trackers, which should be removed and/or replaced with ACRA if really needed. Please resolve this and I think it would be a great addition to F-droid app store! 
![photo_2020-02-15_11-22-36](https://user-images.githubusercontent.com/59568034/74591761-8d641000-5012-11ea-8aa6-61e4d92dc844.jpg)

**Comment by vdbhb59** [comment]
Agreed. It should have trackers removed for FOSS version. OOC, why do you have trackers? This is odd.
